components/nexus-repository/src/main/java/org/sonatype/nexus/repository/group/GroupFacetImpl.java

```diff
@@ -44,11 +44,26 @@
     extends FacetSupport
     implements GroupFacet
 {
-  public static final String CONFIG_KEY = "group";
-
   private final RepositoryManager repositoryManager;
 
-  private final Set<String> memberNames = Sets.newLinkedHashSet();
+  @VisibleForTesting
+  static final String CONFIG_KEY = "group";
+
+  @VisibleForTesting
+  static class Config
+  {
+    @NotEmpty
+    Set<String> memberNames;
```

Comment:
We many need to use List here, or sort out how jackson can be taught to retain set ordering.